flow-server/src/main/java/com/vaadin/flow/dom/Node.java

```diff
@@ -372,6 +463,21 @@ public N removeChild(Element... children) {
                     THE_CHILDREN_ARRAY_CANNOT_BE_NULL);
         }
 
+        return removeChild(Arrays.asList(children));
+    }
+
+    /**
+     * Removes the given child element(s).
```

Comment:
Not exactly sure, but I guess now that these new APIs are accepting a collection it is fine to just use the plural without parenthesis:
```suggestion
     * Removes the given child elements.
```

Reply:
Collection may contain only one element, so probably that's fine to say "element(s)"